src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -200,6 +195,40 @@ public PersistentDataStore() {
     return activities;
   }
 
+  public List<Setting> loadSettings()throws PersistentDataStoreException{
+      Map<Setting.SettingType, Setting> settingMap = new HashMap<>();
+      settingMap.put(Setting.SettingType.COLORS ,  new Setting(Setting.SettingType.COLORS, "white")));
+      settingMap.put(Setting.SettingType.FONT_SIZE, new Setting(Setting.SettingType.FONT_SIZE, "normal"));
+
+      List<Setting> setting = new ArrayList<>();
+    Query query = new Query("settings");
+    PreparedQuery results = datastore.prepare(query);
+
+      for (Entity entity : results.asIterable()) {
+        try {
+            Setting settings = settingMap.get(Setting.SettingType.getSettingType((String) entity.getProperty("type")));
```

Comment:
setting*

Reply:
We should pull type out into a variable here and do a null check and print the error.

Reply:
Done